chain/chain/src/chain.rs

```diff
@@ -391,6 +396,60 @@ pub fn check_known(
     check_known_store(chain, block_hash)
 }
 
+// Cache with the mappint from CryptoHash (blocks, chunks) to the number milliseconds that it took to process them.
+// Used only for debugging purposes.
+pub static CRYPTO_HASH_TIMER_RESULTS: Lazy<Mutex<LruCache<CryptoHash, u64>>> =
+    Lazy::new(|| Mutex::new(LruCache::new(10000)));
```

Comment:
Can we make this a field of some struct, rather than a global variable? Global state is bad: it's easy to add, but hard to remove. For diagnostics use-case this is usually acceptable, but best avoided anyway.

Reply:
Yes, I'd also have liked to avoid global variable - but I don't see a good way around it right now (unless we want to push this object through couple layers).

As you've mentioned - this is for diagnostics, so it should be ok.